mocha.el

```diff
@@ -76,6 +77,16 @@ From http://benhollis.net/blog/2015/12/20/nodejs-stack-traces-in-emacs-compilati
       (setq i (+ i 1)))
     dir))
 
+(defun mocha-opts-file (mocha-file)
+  "Return path to mocha.opts file for MOCHA-FILE."
```

Comment:
This is cool. I actually didn't know mocha supported opts files. I've just been using different .dir-locals files when I need different test configs in the same project.